dashboard/app/controllers/lti/v1/account_linking_controller.rb

```diff
@@ -51,6 +51,7 @@ def link_email
             user: existing_user,
             event_name: 'lti_user_signin',
             metadata: metadata,
+            session: session,
```

Comment:
Adding session here, as it may be useful here to map the the statsig_stable_id to a signed-in user if they link to an existing account.

Reply:
Did you check to make sure there isn't a security or privacy issue with logging the whole session? How big is the session? I think we were having issues for a long time because we were storing too much data in Cookies, so we moved it to the session. Now I'm concerned we are dumping a lot more data into this event. I think it's a little risky for "may be useful". If there is a lot of data, this code could instead be specific about which data from the session is being logged.

Reply:
Basically what this functionality does is, if you want to include the stable ID in the event you're logging, you pass in the session to the `log_event` method, and it will extract it. If you don't care about including a stable ID, you can chose to not pass in the session. There are a lot of places where were weren't including the stable ID in the event, because we already have a signed-in user (therefore a user ID). However this account linking step is the case where a user launches from an LMS, and they can chose to continue with creating a new user or link to an existing account. I figured it could be helpful to include the stable ID in this step, which is why I added it.